src/dialects/postgres/schema/tablecompiler.js

```diff
@@ -46,35 +46,35 @@ TableCompiler_PG.prototype.addColumns = function(columns, prefix, colCompilers)
     // alter columns
     for (const col  of colCompilers) {
       const quotedTableName = this.tableName();
-      const colName = col.getColumnName();
+      const colName = this.client.wrapIdentifier(col.getColumnName(), col.columnBuilder.queryContext());
```

Comment:
maybe this should be something like `this.formatter.wrap(col.getColumnName())`

Reply:
I did try that when running tests as well - I'll use that method instead.